PF/PF-Assignment2/DiamondPyramid.java

```diff
@@ -0,0 +1,58 @@
+// This class creates a diamond pattern using modular approach
+public class DiamondPyramid 
+{
+	// Method that joins the spaces and numbers string
+	// and builds the pyramid
+	public static String[] wholePyramid(int max_num)
+	{
+		// checks if the input is valid 
+		String[] invalid = {"invalid input"};
+		if(max_num<=0)
+			return invalid;
+		
+		// creates a string array to store result
+		String[] result = new String[2*max_num-1];
+		
+		// joins spaces and number strings in for of
+		// resultant pyramid
+		for(int i=0;i<max_num;i++)
+		{
+			result[i] = returnSpaces(max_num,i)+returnNumbers(max_num,i)+returnSpaces(max_num,i);
+			result[2*max_num-2-i] = returnSpaces(max_num,i)+returnNumbers(max_num,i)+returnSpaces(max_num,i);
+		}
+		
+		//returns resultant string array
+		return result;
+	}
+	public static String returnSpaces(int max_num, int k)
+	{
+		// creates string to store spaces
+		String spaces = new String();
+		
+		// modifies spaces string
+		for(int i=0;i<max_num-k-1;i++){
```

Comment:
Try to have consistency in the braces style, they must be either on the same line or on next line everywhere.